.storybook/css-reset-decorator.js

```diff
@@ -1,6 +1,6 @@
 import React, { Fragment } from 'react'
 
-import { CssReset } from '../src'
+import { CssReset } from '../src/index.js'
```

Comment:
The `.js` extension is probably superfluous here - doesn't really hurt, but a bit cleaner and easier to refactor as a subdirectory later if you use `../src/index` (bad example for the refactoring case... `./AlertBar/Action` below is better)

Reply:
It looks like you added it intentionally though?

Reply:
I did.

It is to align closer to the ECMAScript spec than to specific module resolver implementations. Node and Webpack have implemented the module resolution algorithm based on [RequireJS](https://requirejs.org/docs/api.html#jsfiles). The ECMAScript Module definition is much stricter and is based on paths instead of module names.

Given the file `./src/index.js` RequireJS allows these to be equivalent:

- `./src/index.js`
- `./src/index`
- `./src`

Whereas in ESM only the `./src/index.js` path would be valid. I prefer aligning with ESM in our libraries.

I think `api-index.js`, `external-api.js`, `main.js`, or `api.js` is perhaps a less confusing name than `index.js` even though it is technically an index in the literal sense.

Reply:
This particular import is in Storybook though so it doesn't matter. 🤷‍♂